db.php

```diff
@@ -1,15 +1,22 @@
 <?php
 
 class Database{
-  public static function connection() {
+
+  public function connection() {
```

Comment:
this function should be static to work with the rest of our code. The static function that i have used to the left is called like this:  Database::connection();

a public function is called with an -> new Database()->connection() which means that the rest of our code will not work with this 'public function connection(). Please change it so that it works with all of the code.